setup.py

```diff
@@ -1,69 +1,55 @@
-"""
-Stripped down from the example at:
-https://github.com/pypa/sampleproject
-"""
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
 
-import os
+"""The setup script."""
 
-# Always prefer setuptools over distutils
 from setuptools import find_packages, setup
 
-here = os.path.abspath(os.path.dirname(__file__))
+with open('README.md') as readme_file:
+    readme = readme_file.read()
 
+with open('HISTORY.md') as history_file:
+    history = history_file.read()
 
-setup(
-    name='btb',
-
-    # Versions should comply with PEP440.  For a discussion on single-sourcing
-    # the version across setup.py and the project code, see
-    # https://packaging.python.org/en/latest/single_source_version.html
-    version='1.0.0',
+requirements = [
```

Comment:
Can we specify minimum requirements instead of exact requirements?

Reply:
Sure. I will update the specification.